### ISSUE TITLE
Files not opening File location from Task manager.

### ISSUE THREAD

**Issue by Trustery** [files-community/Files#3117]
**Describe the bug**
Files not opening File location from Task manager.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open Task Manager
2. Right click on any task
3. Click on 'Open File location'

**Expected behaviour**
Files should open the file location of the task.

**Desktop**
 - OS Version: Windows 10 20H2 19042.746
 - App version: 1.1.0.0

**Log file**
[debug.txt](https://github.com/files-community/Files/files/5826379/debug.txt)

**Comment by yaichenbaum** [comment]
@Trustery Did you follow the [documentation](https://files-community.github.io/docs/#/articles/replace-file-explorer) to set Files as the default file manager?

**Comment by Trustery** [comment]
Yes I did. Files is working for anything else, but when I click on "Open File location" in task manager you can see In the taskbar that Files open briefly but close immediately afterwards.

**Comment by yaichenbaum** [comment]
This may be a limitation with the way Task Manager works, @gave92 would oyu happen to more on this?

**Comment by gave92** [comment]
@yair100 Interesting, task manager tries to start Files with administrator privileges! But that sadly fails :)

![image](https://user-images.githubusercontent.com/9673091/104858387-f6d55880-591e-11eb-98ba-fbe017cec600.png)
